### ISSUE TITLE
Gradle/Maven dependency manager

### ISSUE THREAD

**Issue by marcellodesales** [MachinePublishers/jBrowserDriver#8]
Hi there,

Are you guys planning on providing a Gradle/Maven dependency on Maven Central?

thanks
Marcello

**Comment by hollingsworthd** [comment]
I did set up an account on Maven Central some time ago. But I haven't gotten around to uploading/signing the distribution. I'll update this ticket when it's on Maven Central but I don't have an ETA yet.

FYI, I've found the easiest way to add a local jar in Maven is:

<project ...>
...
  <repositories>
    <repository>
      <id>local-repo</id>
      <url>file://${project.basedir}/lib</url>
    </repository>
  </repositories>
...
  <dependencies>
...
    <dependency>
      <groupId>local</groupId>
      <artifactId>jbrowserdriver</artifactId>
      <version>0.4.4</version>
    </dependency>
...

And then in the same dir as your pom.xml file, create a file structure like this:

/lib/local/jbrowserdriver/0.4.4/jbrowserdriver-0.4.4.jar
